packages/shared/routes/dashboard/settings/views/Security.svelte

```diff
@@ -196,7 +197,7 @@
                 },
             })
         } else if (newPincode !== confirmedPincode) {
-            newPincodeError = locale('error.pincode.match')
+            confirmationPincodeError = locale('error.pincode.match')
```

Comment:
What do you think about also adding an error for the setup in pin confirmation `routes/setup/protect/views/Pin.svelte`? Right now the proceed button is just disabled when the pins dont match, but maybe it is better displaying an error?

Reply:
Sounds like a good idea, I think there was an issue someone raised like that. Let me add to this PR

Reply:
Done